Quorum/check_proposal.py

```diff
@@ -106,9 +111,12 @@ def main() -> None:
     if config_data:
         # Multi-task mode using JSON configuration
         for customer, chain_info in config_data.items():
+            pp.pprint('Run Preparation', pp.Colors.INFO)
             ground_truth_config = ConfigLoader.load_customer_config(customer)
             GitManager(customer, ground_truth_config).clone_or_update()
             price_feed_providers = ground_truth_config.get("price_feed_providers", [])
+            pp.pprint('Run Metadata', pp.Colors.INFO)
+            pp.pprint(f'Customer: {customer}\nChains and payloads:\n{chain_info}', pp.Colors.INFO)
```

Comment:
```suggestion
            
            pp.pprint('Run Metadata', pp.Colors.INFO)
            pp.pprint(f'Customer: {customer}\nChains and payloads:\n{chain_info}', pp.Colors.INFO)
```

If we print logic separation, it will be more readable to also have a line separation